app/web/src/newhotness/Explore.vue

```diff
@@ -1041,7 +1051,7 @@ watch(filteredComponents, () => {
   if (mapRef.value && typeof mapRef.value.deselect === "function") {
     mapRef.value.deselect();
   }
-  clearSelection();
+  // clearSelection();
```

Comment:
Filtered components may change when the filter changes or when the components list itself changes. For the first case this is unnecessary, but in the second, not clearing the selection may result in the selected component changing (if the list changes in a way the the selected index points to a different component. I'm not sure whats the best behaviour here but keeping it is the safest in a way